tests/libcxx/expected_results.txt

```diff
@@ -566,48 +524,43 @@ std/utilities/allocator.adaptor/allocator.adaptor.members/construct_pair_rvalue.
 
-# Tests ignore the return value of sto{meow}, triggering nodiscard warnings
-std/strings/string.conversions/stod.pass.cpp FAIL
-std/strings/string.conversions/stof.pass.cpp FAIL
-std/strings/string.conversions/stoi.pass.cpp FAIL
-std/strings/string.conversions/stol.pass.cpp FAIL
-std/strings/string.conversions/stold.pass.cpp FAIL
-std/strings/string.conversions/stoll.pass.cpp FAIL
-std/strings/string.conversions/stoul.pass.cpp FAIL
-std/strings/string.conversions/stoull.pass.cpp FAIL
-
 # We define __cpp_lib_has_unique_object_representations in C++17 mode; test error says it
 # "should not be defined when TEST_HAS_BUILTIN_IDENTIFIER(__has_unique_object_representations) || TEST_GCC_VER >= 700 is not defined!"
-std/language.support/support.limits/support.limits.general/type_traits.version.pass.cpp:1 FAIL
+std/language.support/support.limits/support.limits.general/type_traits.version.compile.pass.cpp FAIL
 
 # Tests expect __cpp_lib_ranges to have the old value 201811L for P0896R4; we define the C++20 value 201911L for P1716R3.
-std/language.support/support.limits/support.limits.general/algorithm.version.pass.cpp FAIL
-std/language.support/support.limits/support.limits.general/functional.version.pass.cpp FAIL
-std/language.support/support.limits/support.limits.general/iterator.version.pass.cpp FAIL
+std/language.support/support.limits/support.limits.general/algorithm.version.compile.pass.cpp FAIL
+std/language.support/support.limits/support.limits.general/functional.version.compile.pass.cpp FAIL
+std/language.support/support.limits/support.limits.general/iterator.version.compile.pass.cpp FAIL
 
 # Test expects __cpp_lib_chrono to have the old value 201611L for P0505R0; we define the C++20 value 201907L for P1466R3.
-std/language.support/support.limits/support.limits.general/chrono.version.pass.cpp FAIL
+std/language.support/support.limits/support.limits.general/chrono.version.compile.pass.cpp FAIL
 
 # We unconditionally define __cpp_lib_addressof_constexpr; test error says it
 # "should not be defined when TEST_HAS_BUILTIN(__builtin_addressof) || TEST_GCC_VER >= 700 is not defined!"
-std/language.support/support.limits/support.limits.general/memory.version.pass.cpp FAIL
+std/language.support/support.limits/support.limits.general/memory.version.compile.pass.cpp FAIL
 
 # libc++ doesn't implement P2231R1 Add further constexpr support for optional/variant
-std/language.support/support.limits/support.limits.general/variant.version.pass.cpp FAIL
+std/language.support/support.limits/support.limits.general/variant.version.compile.pass.cpp FAIL
 
 # Tests non-portable behavior
 std/utilities/format/format.formatter/format.context/format.context/advance_to.pass.cpp FAIL
 std/utilities/format/format.formatter/format.context/format.context/arg.pass.cpp FAIL
 std/utilities/format/format.formatter/format.context/format.context/ctor.pass.cpp FAIL
 std/utilities/format/format.formatter/format.context/format.context/locale.pass.cpp FAIL

```

Comment:
These formatter tests need to create `basic_format_context`s. I believe that if we provide `test_format_context_create` in [`test_format_context.h`](https://github.com/llvm/llvm-project/blob/main/libcxx/test/support/test_format_context.h), some of these tests can work. (Some of these use `std::__format_context_create`. They need to be changed to use `test_format_context_create`.)